workflow-service/src/main/resources/application-local.yml

```diff
@@ -33,3 +33,13 @@ logging:
 # ApplicationProperties class to have type-safe configuration
 # ===================================================================
 
+notification:
+  url: "${NOTIFICATION_SERVICE_URL:http://localhost:8081}"
+  auth:
+    basic:
+      user: test
+      password: test
+
+workflows:
+  mta:
```

Comment:
not sure if this (MATA) should be here, or should be part of workflow-examples/src/main/resources/